TYPO3.Flow/Classes/TYPO3/Flow/Core/Booting/Scripts.php

```diff
@@ -637,6 +637,27 @@ public static function executeCommand($commandIdentifier, array $settings, $outp
     }
 
     /**
+     * Executes the given command as a sub-request to the Flow CLI system without waiting for the output.
+     * 
+     * Note: As the command execution is done in a separate thread potential exceptions or failures will *not* be reported
+     *
+     * @param string $commandIdentifier E.g. typo3.flow:cache:flush
+     * @param array $settings The TYPO3.Flow settings
+     * @param array $commandArguments Command arguments
+     * @return void
+     * @api
+     */
+    public static function executeCommandAsync($commandIdentifier, array $settings, array $commandArguments = array())
+    {
+        $command = self::buildSubprocessCommand($commandIdentifier, $settings, $commandArguments);
+        if (DIRECTORY_SEPARATOR === '/') {
+            exec($command . ' > /dev/null 2>/dev/null &');
+        } else {
+            pclose(popen('START /B CMD /S /C "' . $command . '" > NUL 2 > NUL &', 'r'));
```

Comment:
> does it really make a difference though?

Tested it again and: yes, it does in case of an error:
`START /B CMD /S /C "unknowncommand > test.txt" > NUL 2 > NUL &`
vs
`START /B CMD /S /C "unknowncommand > test.txt" > NUL 2> NUL &`

pushing a follow-up, thx!